srcpkgs/tlpui/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'tlpui'
+pkgname=tlpui
+version=1.5.0.5
+revision=1
+wrksrc="TLPUI-${pkgname}-${version%.*}-${version##*.}"
```

Comment:
I think it might be easier if we versioned as `1.5.0+5` (or some other non-`.` non-`-` char). then it would be a simple `${version/+/-}`